widgets/linechart/internal/axes/scale_test.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 // mustNewYScale returns a new YScale or panics.
-func mustNewYScale(min, max float64, graphHeight, nonZeroDecimals int, mode YScaleMode) *YScale {
-	s, err := NewYScale(min, max, graphHeight, nonZeroDecimals, mode)
+func mustNewYScale(min, max float64, graphHeight, nonZeroDecimals int, mode YScaleMode, formatter valueFormatter) *YScale {
```

Comment:
This might still be work in progress, is the plan to use this and add one test that would set the valueFormatter? Or are we getting good test coverage via the other tests (label and axes)?

Reply:
We are already using it:
https://github.com/mum4k/termdash/pull/188/files#diff-82004fbfa0d341d2a0ce7179c86d5f13R200

Reply:
Thank you, I have somehow missed that.